integration_tests/tests/forseti/controls/scanner.rb

```diff
@@ -0,0 +1,50 @@
+# Copyright 2018 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+require 'json'
+control 'Scanner' do
+
+
+   describe "Run scanner" do
+
+       before :context do
+           command("forseti inventory purge 0").result
+           command("forseti inventory create").result
+           @inventory_id = JSON.parse(command("forseti inventory list").stdout).fetch("id")
+           command("forseti model create --inventory_index_id #{@inventory_id} model_scanner").result
+           command("forseti model use model_scanner").result
+           command("sudo cp /home/ubuntu/forseti-security/configs/forseti_conf_server.yaml /home/ubuntu/forseti-security/configs/forseti_conf_server_backup.yaml").result
+           @modified_yaml = yaml('/home/ubuntu/forseti-security/configs/forseti_conf_server.yaml').params
+           @modified_yaml["scanner"]["scanners"][0]["enabled"] = "true"
```

Comment:
The reference to `[0]` is brittle.  What if the configuration change in the future?  If something is added in front of it, then this test will stop working.  Should do something more flexible: either determine the actual position of `audit_logging` or the first disabled scanner.

Reply:
Updated. PTAL